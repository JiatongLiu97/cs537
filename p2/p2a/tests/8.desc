getnumsyscalls test: pipe(), write(), read(), close(), dup() and child process behavior 
