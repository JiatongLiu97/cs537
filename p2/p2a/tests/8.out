XV6_TEST_OUTPUT child 5
XV6_TEST_OUTPUT parent 6
